pyproject.toml

```diff
@@ -19,3 +19,61 @@ build-backend = "setuptools.build_meta:__legacy__"
 # Uncomment if pyproject.toml worked fine to ensure consistency with flake8
 # line-length = 120
 target-version = ["py37", "py38", "py39", "py310"]
+
+
+[tool.ruff]
+target-version = "py38"
+
+ignore = [
+    # these ignores are from flake8-bugbear; please fix!
+    "B007", "B008", "B017",
+    "B018", # Useless expression
+    "B019", "B020",
+    "B022", # Allow empty context manager
+    "B023", "B024", "B026",
+    "B028", # No explicit `stacklevel` keyword argument found
+    "B027", "B904", "B905",
+    "C408", # C408 ignored because we like the dict keyword argument syntax
```

Comment:
Missing the C419 ignore from flake8, make sure this is rebased on main.

Reply:
Is this a new thing added? I created the branch on Friday. May also be the flake8 to ruff script… did you see anything else missing I should be aware of?

Reply:
That's the only recent PR to touch the flake8 config, I can think of.

Reply:
SG. Thanks!